recruiter/tests.py

```diff
@@ -276,3 +284,14 @@ def test_update_recruiter_account_settings_with_valid_data(example_recruiter, ex
     assert example_recruiter_from_db.phone_number == new_phone_number
     # Test that untouched data hasn't changed
     assert example_recruiter_from_db.company == example_recruiter.company
+
+
+@pytest.mark.django_db
+def test_add_new_comapny(valid_company_data, client):
+    response = client.post("/companyRegister/", data=valid_company_data)
+    assert response.status_code == 302
```

Comment:
Missing check for redirection destination

Reply:
will be fixed.